src/capitalsFirst.js

```diff
@@ -21,7 +21,20 @@
  * @returns {string} - string with uppercase words in front
  */
 function capitalsFirst(str) {
-  // write code here
+  let splitedArray = str.split(' ');
```

Comment:
suggestion: words, or wordsArray instead of splitedArray.

Reply:
done